plugins/ag-charts-generate-example-files/src/executors/generate/generator/transformation-scripts/chart-vanilla-src-parser.ts

```diff
@@ -35,7 +35,7 @@ const chartVariableName = 'chart';
 const optionsVariableName = 'options';
 
 function tsGenerateWithOptionReferences(node, srcFile) {
-    return tsGenerate(node, srcFile).replace(new RegExp(`AgCharts\\.update\\(chart, options\\);?`, 'g'), '');
+    return tsGenerate(node, srcFile).replace(new RegExp(`chart\\.update\\(options\\);?`, 'g'), '');
```

Comment:
packages/ag-charts-website/src/content/docs/bullet-series-test/_examples/5-bullets/main.ts has names like `chartOptions5`. Maybe we could do this:-

```suggestion
    return tsGenerate(node, srcFile).replace(/chart[A-Za-z0-0]*\.update\(options\);?/g, '');
```

Reply:
Thanks - fixed.